src/test/java/org/jabref/model/database/BibDatabaseContextTest.java

```diff
@@ -0,0 +1,68 @@
+package org.jabref.model.database;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.jabref.model.metadata.FileDirectoryPreferences;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertTrue;
+
+public class BibDatabaseContextTest {
+
+    private String currentWorkingDir;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    FileDirectoryPreferences preferences;
+
+    @Before
+    public void setUp() {
+        Map<String, String> mapFieldDirs = new HashMap<>();
+        preferences = new FileDirectoryPreferences("saulius", mapFieldDirs, true);
+        currentWorkingDir = Paths.get(System.getProperty("user.dir")).toString();
```

Comment:
The 'currentWorkingDir' is not intended to hold any created files; actually, these tests should not create any files (I'll double check that).

The  'currentWorkingDir' variable is used to re-create the part of the JabRef internal state that we get when the 'jabref biblio.bib' command is invoked from a command line (on Unix/Linux, but I guess on Windows clones as well). Since the directory is different on various computers that invoke the test, I can not hard-code it into the test but must determine it at run-time.

Should I comment this in the code, or find a better name for the 'currentWorkingDir' variable?